organization/roadmap.md

```diff
@@ -0,0 +1,67 @@
+---
+type: Resource
+---
+
+# Roadmap
+
+This roadmap serves as a model for how we're developing, growing and scaling our organization. It sets out the roles for both the staff and the members of the Foundation for Public Code.
+
+## Phase 0: get the organization founded
+
+Staff of the Foundation for Public Code:
+
+* Build team
+* Map potential members
+* Establish a legal entity
+
+This phase is over when: the organization has a legal entity
+
+## Phase 1: developing founding membership
+
+Staff of the Foundation for Public Code:
+
+* Build a diverse founding member base
+* Develop business cases and case study material
+* Pilot codebase stewardship
+* Research service development
+* Define the value proposition for members
+* Define what founding membership would entail
+
+Founding members of the Foundation for Public Code:
+
+* Become founding members
+* Take co-ownership over the organization
+* Organize the first conference for founding members and stakeholders
+* Build organizational alignment within their organizations and with other members
+* Shape the governance and financial structure of the organization
+
+This phase is over when: when there is a member base willing to commit resources and finances for phase 2
+
+## Phase 2: Building a scalable organization
+
+Staff of the Foundation for Public Code:
+
+* Provide (beta) codebase stewardship (no long term guarantees)
+* Develop scalable operations and communication
+* Establish a clear and convincing value proposition and impact model
+
+Members of the Foundation for Public Code:
+
+* Develop a scalable business and governance model
+* Develop a growth model for the organization
+
+This phase is over when: the members are willing to commit to stage 3 and scaling
+
+## Phase 3: Scale
+
+Staff of the Foundation for Public Code:
+
+* Provide codebase stewardship at scale
+* Grow our member base aggressively
+* Consolidate partner and affiliated organizations
+
+Members of the Foundation for Public Code:
+
+* Govern the organization
+
+This phase is over when: when the mission is fulfilled
```

Comment:
if our mission is: 
To enable public-purpose software and policy that is open and collaborative
then there is no (grammatical) end, we can always keep enabling?

the thing we are enabling (public-purpose software and policy that is open and collaborative everywhere and always)  could be 'fulfilled'.